package.json

```diff
@@ -10,6 +10,7 @@
   "scripts": {
     "start": "./entry.sh",
     "build": "webpack --optimize-minimize",
+    "build-no-optimize": "webpack",
```

Comment:
You could change to the following
```
"build": "webpack"
```
```
npm run build -- --optimize-minimize
```
but I think it's nice to have the default be the one we would usually want.